source/fundamentals/crud/read-operations/skip.txt

```diff
@@ -0,0 +1,82 @@
+=====================
+Skip Returned Results
+=====================
+
+.. default-domain:: mongodb
+
+In this guide, we show you how to skip returned results.
+
+You can skip results using the ``skip()`` method of a 
+:java-docs:`FindIterable </apidocs/mongodb-driver-sync/com/mongodb/client/FindIterable.html#skip(int)>`
+instance or by using the :java-docs:`Aggregates.skip() </apidocs/mongodb-driver-core/com/mongodb/client/model/Aggregates.html#sort(org.bson.conversions.Bson)>` 
+static factory method in an aggregation pipeline. 
+
+The ``skip()`` method takes an integer that specifies how many documents
+to omit from the beginning of the list of documents returned by your
+query. 
+
+You can use the ``skip()`` method of a ``FindIterable`` instance as follows:
+
+.. code-block:: java
+
+   collection.find().skip(2); 
+
+The ``Aggregates.skip()`` method implements the ``Bson`` interface to
+specify skip criteria as an argument.  
+
+You can use the ``Aggregates.sort()`` method as follows:
+
+.. code-block:: java
+
+   import com.mongodb.client.model.Aggregates;
+
+   collection.aggregate(Arrays.asList(Aggregates.skip(2)));
+   
+
+The example in this guide is about a paint store that sells eight
+different colors of paint.  The best colors sell quicker than the other
+colors. One day, a customer comes in and asks what the three
+best-selling colors are.  The following documents are in the
+``paint_inventory`` collection: 
+
+.. code-block:: json
+
+   { "_id": 1, "color": "red", "qty": 5 }
+   { "_id": 2, "color": "purple", "qty": 10 }
+   { "_id": 3, "color": "blue", "qty": 9 }
+   { "_id": 4, "color": "white", "qty": 6 }
+   { "_id": 5, "color": "yellow", "qty": 11 }
+   { "_id": 6, "color": "pink", "qty": 3 }
+   { "_id": 7, "color": "green", "qty": 8 }
+   { "_id": 8, "color": "orange", "qty": 7 }
+
+
+To address the scenario, you query the ``paint_inventory`` collection
+with an empty filter, sort the documents by ``qty`` field and omit the first 5
+results.
+
+.. literalinclude:: /includes/fundamentals/code-snippets/Skip.java
+   :language: java
+   :dedent:
+   :start-after: begin skipExample
+   :end-before: end skipExample
+
+- The ``find()`` method returns all documents.  
+- The ``sort()`` method specifies documents to display from highest to lowest based on the ``qty`` field.
+- The ``skip()`` method specifies to omit the first 5 documents. 
+
+The following shows the output of the query specified above:
+
+.. code-block:: json
+   :copyable: false
+   
+   { "_id": 4, "color": "white", "qty": 6 }
+   { "_id": 1, "color": "red", "qty": 5 }
+   { "_id": 6, "color": "pink", "qty": 3 }
+
+After we run the query, we found the three best-selling colors are pink,
+red, and white. 
+
+.. note::
+   If the value of skip exceeds the number of matched documents for a
+   query, that query returns no documents.
```

Comment:
Nitpick: it wouldn't hurt to also show the syntax of the equivalent aggregation to retrieve these results.